src/main/webapp/js/queue.js

```diff
@@ -0,0 +1,51 @@
+/* exported verifyURLStructure */
+/*
+  Makes sure the stub of the user's URL is valid and will warn user
+  if their link wasn't formatted correctly for the server to parse.
+*/
+function verifyURLStructure(url) {
+  if (!url.includes('https://www.youtube.com/watch?v=') || url.length <= 32) {
```

Comment:
What is the user ends up adding http:// instead? Should we cover that case too?

Reply:
I'm not sure. On one hand, it makes the possible links more universal, but then we run the risk of less security because of http standard having less protection. Thoughts?

Reply:
Ya. This could be optional. But something to think about.

We could modify the URL from http to https to fix the security issue otherwise. 

Optional: Not sure if you used regular expressions yet. But, for detecting URL patterns, it is recommended to use them.

Reply:
I haven't written a regex outside of mathematics, so I just found one built for YouTube validation instead